eng/pipelines/templates/steps/cmake-build.yml

```diff
@@ -15,6 +15,19 @@ steps:
     workingDirectory: build
     displayName: cmake --version
 
+  - script: vcpkg --version
+    workingDirectory: build
+    condition: >-
+      and(
+        succeeded(),
+        not(contains(variables['OSVmImage'], 'macOS'))
+      )
+    displayName: vcpkg --version
+
+  - script: clang --version
+    workingDirectory: build
+    displayName: clang --version
```

Comment:
Similar information comes in the cmake generate logs: https://dev.azure.com/azure-sdk/public/_build/results?buildId=4439567&view=logs&j=838825d2-ae5e-5d4d-85b0-bdc7532c5664&t=28f15008-53a6-5f5b-c0f0-bafb46e462a9&l=200

cmake logs: 15.0.0.15000309

clang --version: 15.0.0 (clang-1500.3.9.4)

Reply:
maybe, i just put it there for clarity , looking for it in the logs , i never knew it was there.